rules/windows/execution_downloaded_url_file.toml

```diff
@@ -35,26 +35,31 @@ sequence by process.entity_id with maxspan=2s
 
 [[rule.threat]]
 framework = "MITRE ATT&CK"
+
 [[rule.threat.technique]]
-id = "T1193"
+id = "T1566"
+name = "Phishing"
+reference = "https://attack.mitre.org/techniques/T1566/"
+[[rule.threat.technique.subtechnique]]
+id = "T1566.001"
 name = "Spearphishing Attachment"
-reference = "https://attack.mitre.org/techniques/T1193/"
+reference = "https://attack.mitre.org/techniques/T1566/001/"
 
-[[rule.threat.technique]]
-id = "T1192"
+[[rule.threat.technique.subtechnique]]
+id = "T1566.002"
 name = "Spearphishing Link"
-reference = "https://attack.mitre.org/techniques/T1192/"
-
-[[rule.threat.technique]]
-id = "T1204"
-name = "User Execution"
-reference = "https://attack.mitre.org/techniques/T1204/"
+reference = "https://attack.mitre.org/techniques/T1566/002/"
 
 [[rule.threat.technique]]
 id = "T1064"
 name = "Scripting"
```

Comment:
deprecated

Reply:
is there a reason Scripting wasn't removed, deprecated in favor of Command and Scripting Interpreter